<launch>
    <node pkg="serial" type="serial_node" name="serial" output="screen">
        <!-- set a private parameter for the node -->
        <param name="pubTopic" type="string" value="odom"/>
        <param name="subTopic" type="string" value="cmd_vel"/>
        <param name="serialName" type="string" value="/dev/ttyUSB1"/>
    </node>

</launch>
